packages/components/src/PieChart/PieChartButton.component.js

```diff
@@ -31,7 +33,7 @@ export function decorateWithOverlay(
 				trigger="click"
 				rootClose
 				placement={overlayPlacement}
-				overlay={<Popover id={overlayId}>{overlayComponent}</Popover>}
+				overlay={<div id={overlayId}>{overlayComponent}</div>}
```

Comment:
There is a Popover component in Coral if we want to use it here.

Reply:
Yes that's true , but i think we don't really need a popover here as overlaytrigger is already some sort of popover so it's like putting popover into your popover 😂